doc/nrf/ug_multi_image.rst

```diff
@@ -134,35 +145,12 @@ The three options are:
 
 You must add these four configuration options to the Kconfig file for your child image, replacing *IMAGENAME* with the (uppercase) name of your child image (as specified in ``add_child_image``).
 
-The following example shows the configuration options for MCUboot:
+This can be done by including the :file:`Kconfig.template.build_strategy` template, as shown below.
```

Comment:
What do we refer to by saying "**this** can be done"? What is "this" in this case? We mean "adding config options to the Kconfig file for your child image"?

Btw, just looking at the sentence above, "You must add these four configuration options..."
What four configuration options?

Reply:
Rephrased.